scripts/dts/new_extract.py

```diff
@@ -145,6 +207,22 @@ def reg_aliases(reg):
 
     if reg.name:
         aliases.append(reg_name_alias(reg))
+
+    return aliases
+
+
+def dev_path_aliases(dev, prop):
+    # Returns a list of aliases (e.g., macro names) to be used for property in the
+    # output. TODO: give example output
```

Comment:
Forgot to update this comment. It does something more specific.

`prop` is a bit vague too.